lineage/figures/figure4.py

```diff
@@ -96,29 +97,29 @@ def figureMaker3(ax, x_Sim, output_Sim, x_Cen, output_Cen, xlabel="Number of Cel
     accuracy_cen_df["State Assignment Accuracy"] = Accuracy_Cen
 
     i = 0
-    plotLineage(regGen(), axes=ax[i], censore=False)
+    plotLineage(regGen(31), axes=ax[i], censore=False)
     ax[i].axis('off')
 
     i = 1
-    plotLineage(cenGen(), axes=ax[i], censore=True)
+    plotLineage(regGen(31), axes=ax[i], censore=False)
     ax[i].axis('off')
 
     i = 2
-    plotLineage(regGen(), axes=ax[i], censore=False)
-    ax[i].axis('off')
```

Comment:
Why are there now two lineages per example (a, c)? I find this confusing, because the x-axis is time. One lineage is enough to make the point.